src/main/java/com/team701/buddymatcher/services/timetable/TimetableService.java

```diff
@@ -2,6 +2,10 @@
 
 import com.team701.buddymatcher.domain.timetable.Timetable;
 
+import java.io.File;
+import java.util.List;
+
 public interface TimetableService {
     Timetable retrieve(String userId);
+    List<String> getCalInfoFromIcs(File file) throws Exception;
```

Comment:
Maybe this could throw more specific exceptions?

Reply:
Yep will have a look into that later!